enable/examples/demo/enable/canvas_demo.py

```diff
@@ -5,9 +5,9 @@
 
 class Demo(DemoFrame):
 
-    def create_component(self):
+    def _create_component(self):
         canvas = Canvas(bgcolor="lightsteelblue", draw_axes=True)
-        from basic_move import Box
+        from enable.examples.demo.enable.basic_move import Box
```

Comment:
@kitchoi I was googling namespace packages in python, and was trying to understand but still am a bit confused.  It seems that doing this, without adding any `__init__.py` files works as we want it to, but I want to confirm I am not missing anything.

Reply:
Does `from .basic_move import Box` not work?

Reply:
The context while running the example in the `etsdemo` GUI application is a little different.The Python path there is the Python path used for running the GUI application, which is not necessarily the same as the Python path used for running the script from the command line. The absolute import removes assumption on the Python path.

This is the block of code relevant while this file is run from the GUI application, the triple-underscore-main thing can be found here:
https://github.com/enthought/traitsui/blob/e913f98d10e96624245c603d856b33ded665d565/ets-demo/etsdemo/app.py#L462-L487

Reply:
> It seems that doing this, without adding any __init__.py files works as we want it to

This was what I expected as well.

Reply:
@jwiggins comment is valid as in it shows the cognitive load required to maintain these examples. See https://github.com/enthought/traitsui/issues/1455

Should https://github.com/enthought/traitsui/issues/1455 be fixed, `from .basic_move import Box` should work (I think).

Reply:
> Should enthought/traitsui#1455 be fixed, `from .basic_move import Box` should work (I think).

That seems like the better solution here

Reply:
Sorry, I was wrong...  I should never guess but try for myself... :( 

`from .basic_move import Box` does not work, if I try to run the script from the command line (I am using master with the `create_component` -> `_create_component` fix:
```
 $ pwd
/Users/kchoi/Work/ETS/enable/examples/enable
$ python canvas_demo.py
...
  File "/Users/kchoi/Work/ETS/enable/enable/example_support.py", line 40, in _component_default
    return self._create_component()
  File "canvas_demo.py", line 10, in _create_component
    from .basic_move import Box
ModuleNotFoundError: No module named '__main__.basic_move'; '__main__' is not a package
```

Reply:
The absolute import here does work on both the etsdemo application and from the command line.
`from basic_move import Box` does not work on etsdemo but will work if https://github.com/enthought/traitsui/issues/1455 is fixed.

@jwiggins Is it okay for us to proceed with the absolute import here to decouple this import from the pending issue?

Reply:
From offline discussion: John says yes. :D